src/app/content/search/utils.ts

```diff
@@ -147,11 +148,20 @@ export const highlightResults = (
     }
 
     const hitHighlights = hit.highlight.visibleContent.map((highlightText, index) => {
-      const highlights = getHighlightRanges(element, highlightText).map((range) => {
-        const highlight = new Highlight(range.nativeRange, {content: range.toString()});
-        highlighter.highlight(highlight);
-        return highlight;
-      });
+      const highlights = getHighlightRanges(element, highlightText)
+        .map((range) => {
+          const highlight = new Highlight(range.nativeRange, {content: range.toString()});
+
+          try {
+            highlighter.highlight(highlight);
+          } catch (e) {
+            Sentry.captureException(e);
+          }
+
+          return highlight;
```

Comment:
I think it's okay to just return highlight after attaching it with `attachHighlight` since it will still get filtered and it avoids additional unnecessary loops.